components/SpotifyInterface/SpotifyInterface.c

```diff
@@ -0,0 +1,355 @@
+#include <stdio.h>
+#include "SpotifyInterface.h"
+#include "SpiffsManger.h"
+// ****************************** Local Variables
+static const char *TAG = "SpotifyTask";
+
+// ****************************** Local Functions
+static void Spotify_GetToken(char *code, size_t SizeOfCode);
+static bool Spotify_IsTokenValid(void);
+static void Spotify_RenewTokenWithRefreshToken(Token_t *token);
+static void Spotify_MainTask(void *pvparameters);
+static bool Spotify_TokenRenew(void);
+static bool CheckTime(bool ExpireFLG);
+// ******************************
+SpotifyPrivateHandler_t PrivateHandler;
+SpotifyInterfaceHandler_t InterfaceHandler;
```

Comment:
it should be Just an empty  local pointer not an other instance of main-heap object. (now we have 2 instance of main-heap object).

Reply:
I solve it